tests/docs-screenshots.spec.ts

```diff
@@ -68,43 +90,646 @@ test("instances", async ({ page }) => {
+    clip: getClipPosition(280, 190, 1165, 640),
+  });
+
+  await deleteVolume(page, volumeName.toLowerCase());
+  await deletePool(page, poolName);
+});
+
+test("LXD - Tutorial folder", async ({ page }) => {
+  await page.setViewportSize({ width: 1440, height: 800 });
+
+  const instance = "Instance1";
+  await gotoURL(page, "/ui/");
+  await page.waitForTimeout(1000);
+  await page.getByRole("button", { name: "default", exact: true }).click();
+  await page.getByRole("button", { name: "Create project" }).click();
+  await page.waitForSelector(`text=Project name`);
+  await page.getByRole("button", { name: "default", exact: true }).click();
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_project.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByRole("button", { name: "default", exact: true }).click();
+  await page.getByRole("link", { name: "Instances", exact: true }).click();
+  await page.getByText("Create instance").click();
+  await page.getByPlaceholder("Enter name").fill("Ubuntu-vm");
+  await page.getByRole("button", { name: "Browse images" }).click();
+  await page
+    .locator("tr")
+    .filter({ hasText: "Ubuntu24.04 LTSnoblealldefaultUbuntuSelect" })
+    .getByRole("button")
+    .click();
+
+  await page.getByLabel("Instance type").click();
+  await page.getByLabel("Instance type").selectOption("VM");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_vm.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByLabel("Instance type").click();
+  await page.getByLabel("Instance type").selectOption("Container");
+  await page.getByPlaceholder("Enter name").fill(instance);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/create_instance.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByText("Resource limits").click();
+  await page.getByText("Resource limits").click();
+  await page
+    .getByRole("row", { name: "Exposed CPU limit" })
+    .getByRole("button")
+    .click();
+  await page.getByPlaceholder("Number of exposed cores").fill("1");
+  await page
+    .getByRole("row", { name: "Memory limit Usage limit" })
+    .getByRole("button")
+    .click();
+  await page.getByPlaceholder("Enter value").fill("4");
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/resource_limits.png",
+    clip: getClipPosition(0, 0, 1440, 760),
+  });
+
+  await page.getByRole("button", { name: "Create and start" }).click();
+  await page.getByTestId("notification-close-button").click();
+
+  await page
+    .getByRole("row", {
+      name: `Select ${instance} default Name Type Description IPv4 Status Actions`,
+    })
+    .getByLabel("Name")
+    .click();
+  await page.waitForSelector(`text=Instance summary`);
+  await page.screenshot({
+    path: "tests/screenshots/doc/images/tutorial/instance_summary.png",
```

Comment:
The info box should be closed on this screen

<img width="1862" height="727" alt="image" src="https://github.com/user-attachments/assets/74532705-d2cb-42cd-baf0-769bf3839ff9" />